facebook/delphiFacebook/R/variables.R

```diff
@@ -69,6 +71,37 @@ code_activities <- function(input_data) {
     input_data$a_large_event_1d <- NA
     input_data$a_public_transit_1d <- NA
   }
+  
+  if ("C13b" %in% names(input_data)) {
+    # introduced in wave 10 as "indoors" activities version of C13
+    activities <- split_options(input_data$C13b)
+    
+    input_data$a_work_outside_home_indoors_1d <- is_selected(activities, "1")
+    input_data$a_shop_indoors_1d <- is_selected(activities, "2")
+    input_data$a_restaurant_indoors_1d <- is_selected(activities, "3")
+    input_data$a_spent_time_indoors_1d <- is_selected(activities, "4")
+    input_data$a_large_event_indoors_1d <- is_selected(activities, "5")
+    
+    transit_responses <- is_selected(activities, "6")
+    if ( is.null(input_data$a_public_transit_1d) ) {
+      input_data$a_public_transit_1d <- transit_responses
+    } else {
+      # Combine with C13 responses
+      input_data$a_public_transit_1d <- coalesce(input_data$a_public_transit_1d, transit_responses)
+    }
```

Comment:
`code_activities()` is only ever called on one wave at a time, though that's not actually documented. (You can see `code_mental_health()` has an assert to enforce that.) Shouldn't it then be impossible for `a_public_transit_1d` to already have data in it?

Reply:
Though if we rely on this fact, we should ensure there's an assert or documentation to enforce it.

Reply:
I removed this bit of logic and added the `assert`.